test-app/tests/integration/components/amount-input/component-test.ts

```diff
@@ -45,14 +60,14 @@ module('Integration | Component | amount-input', function (hooks) {
     assert.dom('input').hasAttribute('max', '10');
   });
 
-  test('uses named property declared (even if undefined) instead of defaults', async function (assert) {
-    await render(hbs`
+  test('uses named property declared (even if undefined) instead of defaults', async function (this: TestContext, assert) {
+    await render<TestContext>(hbs`
       <AmountInput
-        @currency={{this.isUndefined}}
-        @numberOfDecimal={{this.isUndefined}}
-        @placeholder={{this.isUndefined}}
-        @step={{this.isUndefined}}
-        @inputId={{this.isUndefined}}
+        @currency={{undefined}}
+        @numberOfDecimal={{undefined}}
+        @placeholder={{undefined}}
+        @step={{undefined}}
+        @inputId={{undefined}}
```

Comment:
Couldn't we avoid passing them altogether?

```suggestion
```

Reply:
There is another [test](https://github.com/qonto/ember-amount-input/blob/8d0b5dc1c60336d3e94c905c4695fdb9b470fe51/test-app/tests/integration/components/amount-input/component-test.ts#L86-L102) that does what you suggest, but the purpose and the output are not the same